src/sparseml/optim/helpers.py

```diff
@@ -195,6 +195,68 @@ def parse_recipe_variables(
     return recipe_variables
 
 
+def _update_staged_container_value(var_key, var_value, container):
```

Comment:
let's rename `_update_staged_container_value` to `_update_staged_recipe_variable` as container is a little ambiguous outside of the scope of the functions